client.go

```diff
@@ -66,9 +66,11 @@ func Connect(endpoints string, tlsConfig *tls.Config) (*OvsdbClient, error) {
 			return nil, err
 		}
 		var host string
-		endPointStr := strings.Split(endpoint, ":")
-		if len(endPointStr) > 2 {
-			host = fmt.Sprintf("%s:%s", endPointStr[1], endPointStr[2])
+		// Strip off the leading protocol. "tcp:127.0.0.1:6640" should
+		// be converted to "127.0.0.1:6640"
+		endPointStr := strings.SplitN(endpoint, ":", 2)
+		if len(endPointStr) > 1 {
+			host = endPointStr[1]
 			if len(host) == 0 {
 				host = defaultTCPAddress
```

Comment:
I left the check for empty `host` as it was, but I think that check should be outside of the check for `endPointStr` length.